### PR TITLE
Merge feature/realtime2022 branch into support/HAFS

### PR DESCRIPTION
**Pull request by BinLiu-NOAA**
Merge feature/realtime2022 branch into support/HAFS, which will be used by HAFS application level develop branch.

### CODE REVIEW

**Comment by BinLiu-NOAA** [comment]
@ghassan-alaka, @lgramer, or @AndrewHazelton, need at least one review approval to merge this PR. Thanks!

**Review by ghassan-alaka — APPROVED** [review]
All changes appear to be appropriate. As long as any differences in `feature/realtime2022` supersede those in `support/HAFS` during the merge, everything should be fine. I would like @AndrewHazelton or @lgramer can test the updated support/HAFS branch within the HAFS workflow after the merge.

**Pull request merged**
